ethereumClassic-cpp-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,203 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
+      "type": "string",
+      "maxLength": 11,
+      "metadata": {
+        "description": "Name prefix of the Storage Account"
+      }
+    },
+    "vmDnsPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "This is the unique DNS name of the for the public IP for your VM"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "This is the the username you wish to assign to your VMs admin account"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "This is the the password you wish to assign to your VMs admin account"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D1",
+      "allowedValues": [
```

Comment:
There is not really an "upper limit" in terms of system requirements, but there is a diminishing return in the higher price VMs as the system is designed to work on consumer hardware. The nodes of the system have a small enough footprint to run on arm systems. The current system of consensus makes it unlikely that the VM will be used as a mining node. The series "A", "D", and "F" are the most practical for a hosted node. I will update based on: https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-windows-sizes to include F and G series